repaygithub/cactus
# CACTUS- 281 ::  TypeScript Linting

Ticket: https://repayonline.atlassian.net/browse/CACTUS-281

I would definitely recommend just skimming through the actual changes, as they are extensive.

### Testing
1. Check out this branch and run `yarn install`.
2. Run `yarn cleanup && yarn build` at the root to clean & rebuild all modules.
3. Run `yarn test` at the root to:
  - Test types for all modules
  - Run all tests (including integration)
  - Run the linter
4. Make sure that you don't encounter any errors during the tests, type tests, or linting.

If you have any questions, just let me know!
